sandboxed_containers/understanding-sandboxed-containers.adoc

```diff
@@ -0,0 +1,28 @@
+[id="understanding-sandboxed-containers"]
+= Understanding OpenShift sandboxed containers
+include::modules/common-attributes.adoc[]
+
+:context: understanding-sandboxed-container
+
+toc::[]
+
+[role="_abstract"]
+
+OpenShift sandboxed containers support for Red Hat OpenShift provides users seeking additional workload isolation a built-in support for running Kata containers as an additional optional runtime on OpenShift Container Platform. This is particularly useful for users who are wanting to perform the following tasks.
```

Comment:
fwiw, _kata containers_ is an Open Container Initiative (OCI) Runtime for use with CRI-O. Could be of interest to someone completely new to this.

Reply:
Thanks Jason. I will bring this up with the PMs.